package.json

```diff
@@ -46,8 +47,19 @@
   "engines": {
     "node": ">= 10.0.0"
   },
+  "nyc": {
+    "include": [
+      "*.js",
+      "lib/**/*.js"
+    ],
+    "extension": [
+      ".js"
```

Comment:
This is to keep ts-node and nyc from fighting.

If anyone gets ideas of making node-config into an ESM module we will have to deal with #787 first